lib/fail.js

```diff
@@ -23,14 +23,19 @@ export default async function fail(pluginConfig, context, { Octokit }) {
     githubApiPathPrefix,
     githubApiUrl,
     proxy,
-    failComment,
     failTitle,
+    failComment,
+    failCommentCondition,
     labels,
     assignees,
   } = resolveConfig(pluginConfig, context);
 
   if (failComment === false || failTitle === false) {
```

Comment:
Hello there @fgreinacher, @JonasSchubert 

On another note, deprecation of the `failTitle`, what does this mean for `failCommentCondition`??

Considering the `failTitle` is used as value for issue title in the api request to create an issue for failure, its absence is a problem when `failCommentCondition` is `true` and `failTitle` is `false`.

My point is `failCommentCondition` (in our current implementation) doesn't look like it can be used on it's own without having to enable both `failComment` and `failTitle` (kindly correct me if I perceive this wrongly). Considering the established implementation that uses `lodash.template`, are we looking at the possibility of incorporating the fail comment `title` and `body` inside `failCommentCondition` using templates??? and of course split'em back out later???

Is this the plan for that future MAJOR release? 🤔

Reply:
No worries, the plan is only to remove support for disabling comments by setting successComment, failComment, or failTitle to false. It's totally fine to leave them undefined or set them to any other value.